fluid-project/handbook.floeproject.org
# fix: Add UIO contrast theme support to banner

## Description

This update adds UIO contrast theme support to the banner image and ILDH logos. It also improves the scaling of the banner image to better match the design, and fixes a bug that kept the yellow circle from appearing.

## Steps to test

1. Go to the Home page
2. Click on "+ show preferences"
3. Select a contrast theme from the "Contrast" panel
4. Resize the window to large and narrow widths to see all scaling possibilities
5. Go to any other page
6. Resize the window to large and narrow widths to see all scaling possibilities

**Expected behavior:** The banner image should respond to the contrast theme and show outlines of the circles in the foreground colour, and the scaling should not look broken.

The four defined scaling conditions are:
- Home page wide (48 rem and up)
- Home page narrow (under 48 rem)
- Other pages wide
- Other pages narrow

## Additional information

N/A

## Related issues

Resolves #62
